tests/bluetooth/bsim_bt/edtt_ble_test_app/gatt_test_app/prj.conf

```diff
@@ -24,8 +24,10 @@ CONFIG_BT_L2CAP_RX_MTU=512
 CONFIG_BT_L2CAP_TX_MTU=512
 CONFIG_BT_CTLR_RX_BUFFERS=3
 CONFIG_BT_DEBUG_LOG=y
+CONFIG_BT_EATT=n
+
 #
 # To make DEVICE Name writable...
 #
 CONFIG_BT_DEVICE_NAME_DYNAMIC=y
-CONFIG_LOG_MAX_LEVEL=2
+CONFIG_LOG_MAX_LEVEL=4
```

Comment:
Why change the default log level?

Reply:
This was for debugging, there is very little information printed with level 2 not sure why is was done like that.

Reply:
Fixed.